invoke-ai/InvokeAI
# [WIP] Generation backend rewrite

## Summary

Rewriting of generation backend.
Done as draft-suggestion, so any suggestions or patches are welcome.
I think main idea of code is that backend less hardcoded and know less about things that it working with(more abstract).
Also fixes moment, that controlnet not handling properly regional prompts.

## Related Issues / Discussions

None

## QA Instructions

Currently old generation code not removed, so you can patch if in `latents_from_embeddings` and run old logic to check/compare.

## Merge Plan

Remove old backend code, which currently remains for testing.

## Checklist


@dunkeroni @RyanJDick @lstein
